docs/k0s-in-docker.md

```diff
@@ -4,7 +4,7 @@ We publish a k0s container image with every release. By default, we run both con
 
 You can run your own k0s-in-docker easily with:
 ```sh
-docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.pkg.github.com/k0sproject/k0s/k0s:<version>
+docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.io/k0sproject/k0s:latest
```

Comment:
What is the reason for this change?

Reply:
Docker Hub images pull does not need and auth whereas Github registry does always need auth. Easier for the users